stdlib/string.gr

```diff
@@ -351,17 +415,16 @@ export let rec implode = (arr: Array<Char>) => {
   let mut stringByteLength = 0n
 
   for (let mut i = 0n; i < arrLength; i += 1n) {
-    let char = WasmI32.load(arr + (i << 2n), 8n)
-    let byte = WasmI32.load8U(char, 4n)
+    let usv = WasmI32.load(arr + (i << 2n), 8n) >>> 3n
```

Comment:
There's a lot going on here that makes it unreadable for someone without deep knowledge of Grain's internals.
A function for untagging the char and one for reading the array element would make it self documenting. (at the price of the function call, if it's not inlined, of course)

Reply:
Whoops, I meant to add a comment to this.

Reply:
Should be reable now